pkg/products/monitoringcommon/alertmanagerConfig_test.go

```diff
@@ -116,14 +116,11 @@ func TestReconciler_reconcileAlertManagerSecrets(t *testing.T) {
 		},
 		Type: corev1.SecretTypeOpaque,
 	}
-	alertmanagerRoute := &v1.Route{
+	alertmanagerService := &corev1.Service{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      alertManagerRouteName,
+			Name:      alertManagerServiceName,
```

Comment:
Should the name here be `mockAlertManagerServiceName`? If not we should remove that [const](https://github.com/integr8ly/integreatly-operator/pull/3238/files#diff-a0492f0c6f3f90d001065aab6bef7c3084e210616030e84ed72f7c1a3a14e608R33) because it's not being used otherwise. 
```suggestion
			Name:      mockAlertManagerServiceName,
```

Reply:
Done, I've removed the const